azure-pipelines.Windows.yml

```diff
@@ -0,0 +1,16 @@
+trigger:
+  batch: true
+  branches:
+    # blacklist
+    exclude:
+    - gh-pages
+
+name: $(Year:yyyy).$(Month).$(DayOfMonth)$(Rev:.r)
+
+jobs:
+- template: build/azure-pipelines.job.template.yml
+  parameters:
+    name: Windows
+    vmImage: 'vs2017-win2016'
+    scriptFileName: .\build.ps1
+    timeoutInMinutes: 300
```

Comment:
why the `timeoutInMinutes` is 300 for Windows when the default value in the template is 120?

Reply:
I need to remove this, but I am still investigating where deadlock is. Usually, it takes 40 minutes to complete build with all test suites on Windows, and in some cases 3 hours isn't enough to finish the build. Only on Windows. macOS and Linux builds are stable.